index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Title</title>
+</head>
+<body>
+    <header class="navigation">
+        <div class="header__top">
+          <div class="header__logo">
+            <img src="/img/camera__logo.png" alt="camera logo" />
+          </div>
+  
+          <div class="nav__main">
+            <nav class="nav__menu">
+              <ul class="nav__item">
+                <li class="nav_list"><a href="#" class="nav__link">HOME</a></li>
+                <li class="nav_list">
+                  <a href="#" class="nav__link">PRODUCTS</a>
+                </li>
+                <li class="nav_list">
+                  <a href="#" class="nav__link">TESTIMONIALS</a>
+                </li>
+                <li class="nav_list"><a href="#" class="nav__link">ABOUT</a></li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+      <section class="home">
```

Comment:
- [ ] Home sectioningizni bo'yi ekranizni to'liq to'ldirib tursin. O'zimni noutbukimda ochsam sal pasda joy qolyapti.

Hozir landing pageda websaytingizda ikkita narsa bor. 
1. Navbar
2. Section home

shuni hisobiga bo'yini qo'yishda `vh` bn ishlatsak yaxshi. 
misol uchun 

> .nav {
> height: 10vh;
> }
> 
> .home {
> height 90vh;
> }